js/binance.js

```diff
@@ -2244,6 +2244,45 @@ module.exports = class binance extends Exchange {
         return this.parseTickers (response, symbols);
     }
 
+    async fetchL1OrderBooks (symbols = undefined, params = {}) {
+        /**
+         * @method
+         * @name binance#fetchL1OrderBooks
+         * @description fetches the bid and ask price and volume for multiple markets
+         * @param {[str]|undefined} symbols unified symbols of the markets to fetch the bids and asks for, all markets are returned if not assigned
+         * @param {dict} params extra parameters specific to the binance api endpoint
+         * @returns {dict} an array of [ticker structures]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        await this.loadMarkets ();
+        const defaultType = this.safeString2 (this.options, 'fetchBidsAsks', 'defaultType', 'spot');
```

Comment:
`fetchL1OrderBooks` instead of `fetchBidsAsks` ?

Reply:
right, it was renamed, forgot to change it.